app/services/save_provider_user.rb

```diff
@@ -61,7 +58,7 @@ def send_permissions_granted_email
   end
 
   def new_provider_permissions
-    @new_provider_permissions ||= provider_permissions - provider_user.provider_permissions
+    @new_provider_permissions ||= provider_permissions - @old_provider_permissions
```

Comment:
can you add `old_provider_permissions` to `attr_reader` so we don't need to use `@`?

Reply:
sorted